packages/orchestration/README.md

```diff
@@ -4,14 +4,17 @@ This package incorporates generative AI orchestration capabilities into your AI
 
 ## Table of Contents
 
-1. [Installation](#installation)
-2. [Prerequisites](#prerequisites)
-3. [Orchestration Service](#orchestration-service)
-4. [Usage](#usage)
-   - [Templating](#templating)
-   - [Content Filtering](#content-filtering)
-5. [Support, Feedback, Contribution](#support-feedback-contribution)
-6. [License](#license)
+- [@sap-ai-sdk/orchestration](#sap-ai-sdkorchestration)
+  - [Table of Contents](#table-of-contents)
```

Comment:
[pp] I remember during the GA release, we excluded the toc by using `###` for `Table of Contents`

Reply:
This was accidentally changed by some VS code plugin. Reverted it now